src/backend/joanie/core/filters/admin.py

```diff
@@ -0,0 +1,90 @@
+"""
+Admin API Resource Filters
+"""
+from typing import List
+
+from django.db.models import Q
+
+from django_filters import rest_framework as filters
+
+from joanie.core import models
+
+
+class OrganizationAdminViewSetFilter(filters.FilterSet):
+    """
+    OrganizationAdminFilter allows to filter this resource with a search for code and title.
+    """
+
+    search = filters.CharFilter(method="filter_search")
+
+    def filter_search(self, queryset, _name, value):
+        """
+        construct the full expression for search query param.
+        """
+        return queryset.filter(
+            Q(code__icontains=value) | Q(translations__title__icontains=value)
```

Comment:
Are you sure using title directly is not working ?
```suggestion
            Q(code__icontains=value) | Q(title__icontains=value)
```

As this field is a translated one, I wonder what happens if you provide a value in a language and the active locale does not correspond. Can you write a test about that ?

E.g : 
Organization:
French title is : Université 1
English title is : University 1

If you provide `search=University%201` query parameter with a header `Accept-Language` sets to `FR-fr` does it find the organization ?